tests/package.json

```diff
@@ -9,11 +9,11 @@
   "author": "",
   "license": "ISC",
   "devDependencies": {
-    "@renku/notebooks-cypress-tests": "^0.0.7",
+    "@renku/notebooks-cypress-tests": "github:leafty/notebooks-cypress-tests#gh-package-11",
```

Comment:
The comment about using the regular version refers to this... Not to the removal.

Reply:
aha! thanks, that makes more sense now